cmd/mysterium_server/command_run/command.go

```diff
@@ -45,11 +47,28 @@ func (cmd *CommandRun) Run(options CommandOptions) (err error) {
 		return err
 	}
 
+	proposal := service_discovery.NewServiceProposal(
+		providerId,
+		nats.NewContact(providerId),
+	)
+
+	sessionResponseHandler := vpn_session.CreateResponseHandler{
+		ProposalId: proposal.Id,
+		SessionManager: &session.Manager{
+			Generator: &session.Generator{},
+		},
+		ClientConfigFactory: func() *openvpn.ClientConfig {
+			return openvpn.NewClientConfig(
+				vpnServerIp,
+				options.DirectoryConfig+"/ca.crt",
+				options.DirectoryConfig+"/client.crt",
+				options.DirectoryConfig+"/client.key",
+				options.DirectoryConfig+"/ta.key",
+			)
+		},
+	}
 	handleDialog := func(sender communication.Sender, receiver communication.Receiver) {
-		receiver.Respond(communication.GET_CONNECTION_CONFIG, func(request string) (response string) {
-			config, _ := buildVpnClientConfig(vpnServerIp, options.DirectoryConfig)
-			return config
-		})
+		receiver.Respond(communication.SESSION_CREATE, sessionResponseHandler.Handle)
```

Comment:
spacy spaces

Reply:
It seems so on github, IDE shows it fine.